middlewares/validators/levels.js

```diff
@@ -0,0 +1,20 @@
+const Joi = require("joi");
+
+const validateLevelBody = async (req, res, next) => {
+  const schema = Joi.object({
+    name: Joi.string().trim().required(),
+    levelnumber: Joi.number().integer().min(0).required(),
+  });
+
+  try {
```

Comment:
Add Joi part in a try block.

Reply:
all other files have validation in this format.